spec/disposable-spec.coffee

```diff
@@ -20,3 +20,13 @@ describe "Disposable", ->
 
     disposable.dispose()
     expect(disposalAction.callCount).toBe 1
+
+  describe "::isDisposable(object)", ->
```

Comment:
Usually we don't denote these class methods w/ the `::`, because that's coffee-script notation for referring to properties on a prototype.